### PR TITLE
RonDB-276: Single setup integration tests

### PR DESCRIPTION
**Pull request by olapiv**
Based on PR #239

### CODE REVIEW

File: storage/ndb/rest-server/rest-api-server/internal/config/constants.go [hunk 1]
```diff
@@ -21,22 +21,26 @@ import "hopsworks.ai/rdrs/version"
 
 const API_KEY_NAME = "X-API-KEY"
 
+// TODO: What's a "PP"?
```
Comment (smkniazi) [inline storage/ndb/rest-server/rest-api-server/internal/config/constants.go]: it is "path prefix"

**Review by smkniazi — DISMISSED** [review]

**Comment by smkniazi** [comment]
Please put back the log call back functions

**Comment by olapiv** [comment]
@smkniazi I didn't change the log directory in this PR?

**Review by smkniazi — APPROVED** [review]
excellent job. sorry for reverting the review before, I misunderstood the build issue. it was a problem with my env

**Pull request merged**